src/typegraph/python/typegraph/utils.py

```
@@ -89,3 +90,30 @@ def freeze_tg_output(
+    def __init__(self, source: str, deps: List[str] = []):
+        self.source = source
+        self.deps = deps
+
+    def import_(self, name: str):
+        return ResolvedModule(module=self.source, deps=self.deps, func_name=name)
+
+
+def resolve_module_params(
+    module: Union[str, ResolvedModule],
+    name: Optional[str] = None,
+    deps: List[str] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)